lib/govuk_app_config/govuk_i18n.rb

```diff
@@ -0,0 +1,36 @@
+module GovukI18n
```

Comment:
Could the name be changed so this is clearer that it relates to only the pluralisation rules, maybe something like `GovukI18nPlurals`?  Since this only returns a hash, it can't be expanded to any other I18n uses, so the name could potentially be misleading.

Reply:
I think I'll turn it into a `plurals` method so we can expand this in the future